### DISCUSSION TITLE
Clear option for Multiselect Filter

### DISCUSSION THREAD

**Discussion by gokul-shashidhar** [primefaces/community#1314] [PrimeVue]
Hey Folks,

I was hoping if we can have a clear button at the end of the input for Multiselect component with the filter option. It would be a great user experience if they could clear out the input and start over. 
A good example would be similar to github's search bar,  or even slack's search field which has a "clear" at the end. 
![Screenshot 2024-02-28 at 12 05 35](https://github.com/primefaces/community/assets/108738946/6c086a3e-1493-4c1e-8b42-e1d574d8e0b8)

Is it possible to have this or maybe contribute to the repository to include this feature behind a prop?

**Comment by hzulqarnain-coder** [comment]
I also need this.Would be a great addition to the UX.
